### DISCUSSION TITLE
API Lambda Size Stats

### DISCUSSION THREAD

**Discussion by followbl** [vercel/next.js#13852] [Ideas]
Currently the only way to easily know if your api lambda size is over the 50mb limit is deploying via vercel. A great feature would be able to see these lambda sizes via `next build` (very similar to the  page lambdas) so we can avoid having to deploy on Vercel - just to see if we've hit this limit or not. 

Would also in general be good to review the size of each individual lambda.

**Comment by pleaseshutup** [comment]
I am with you. I have been stuck for 4 days unable to deploy due to a lambda being 52mb. I am at a loss as to why it needs to be that big. I need to know what is in that zip file and how I can instruct the build to reduce that by removing anything unnecessary.

**Comment by stnwk** [comment]
Same here, it would help **a lot** if we were able to inspect a failed deployment's output `.next` directory
